lib/BackgroundJob/SyncJob.php

```diff
@@ -33,14 +35,17 @@ class SyncJob extends TimedJob {
 	private LoggerInterface $logger;
 	private IJobList $jobList;
 
-	public function __construct(ITimeFactory $time,
+	public function __construct(
+		ITimeFactory $time,
 		IUserManager $userManager,
 		AccountService $accountService,
 		MailboxSync $mailboxSync,
 		ImapToDbSynchronizer $syncService,
 		LoggerInterface $logger,
 		IJobList $jobList,
-		IConfig $config) {
+		IConfig $config,
+		private IUserConfig $userConfig,
```

Comment:
IUserConfig is new in 31. 
Use IConfig.getUserValue instread.

Reply:
Done